cdap-ui/app/features/admin/admin.less

```diff
@@ -2,6 +2,9 @@
 
 body.theme-cdap.state-admin {
   main.container {
+    .system-preferences {
+      margin-top: 25px;
```

Comment:
Can we use ```<br/>``` tag? When we want to style the padding later it will become a custom fix just for this. I believe we have been using br tag elsewhere when we need a new line (separation of elements vertically)